Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -136,8 +142,10 @@ public override void OnPreSceneQuery()
             if (newClosestTouchable != null)
             {
                 // Build ray (poke from in front to the back of the pointer position)
-                Vector3 start = Position + touchableDistance * closestNormal;
-                Vector3 end = Position - touchableDistance * closestNormal;
+                var lengthOfPointerRay = newClosestTouchable is NearInteractionTouchableVolume ?
```

Comment:
nit:add comment here to better explain the ?: operation. That if volume build a really long raycast